spec/index.html

```diff
@@ -317,7 +463,20 @@ <h2>Changes between publication as Note and RDF 1.1 Recommendation</h2>
   <h2>Changes between RDF 1.1 and RDF 1.2</h2>
 
   <ul>
-    
+    <li>N-Quads is now described as an extension of N-Triples [[RDF12-N-TRIPLES]].</li>
+    <li>Add <a href="#canonical-quads" class="sectionRef"></a> to define the canonical form of N-Quads.</li>
+    <li>Better align the use of white space and comments with [[RDF12-TURTLE]].</li>
+    <li>Removed language about white space use between terminals that would otherwise
+      be (mis-)recognized, is this can't happen in N-Triples.</li>
+    <li>Clarify the use of blank lines, including those composed of white space
+      and/or comments.
+      Comments can appear at the end of a triple before the newline as
+      was already evident from <a href="#ex-comments"></a>.</li>
+    <li>Create separate subsections of <a href="#n-quads-grammar"></a>
+      for <a href="#sec-grammar-ws">White space</a> and
+      <a href="#sec-grammar-comments">Comments</a>,
+      better mirroring [[RDF12-TURTLE]].</li>
+    <li>A valid N-Quads document must end with an EOL <code>U+000A</code>.</li>
```

Comment:
I believe it applies only for *canonical* N-Quads documents. If not, the grammar should be updated.

Reply:
Yes, I think you're correct. And, it should probably apply to N-Triples C14N as well. I'll revert a couple of changes such as this, that were not restricted to C14N.